.github/workflows/ci.yml

```diff
@@ -131,28 +131,6 @@ jobs:
         working-directory: pop-api
         run: cargo test --release --locked --all-features
 
-  api-integration-tests-devnet:
```

Comment:
no longer necessary as it's included in the regular `cargo test`

Reply:
this comment is outdated. We introduced a new task that does api-integration-tests but only requires one test for devenet & testnet.